### PR TITLE
Scroll to bottom button style changes to match theme.

### PR DESCRIPTION
**Pull request by pprucia**
### Contributor checklist

- [x] I am following the [Code Style Guidelines](https://github.com/WhisperSystems/Signal-Android/wiki/Code-Style-Guidelines)
- [x] I have tested my contribution on these devices:
 * Samsung Galaxy Nexus, Android 4.3
 * Virtual Nexus 5, Android 7.1
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)
- [x] I have made the choice whether I want the [BitHub reward](https://github.com/WhisperSystems/Signal-Android/wiki/BitHub-Rewards) or not by omitting or adding the word `FREEBIE` in the commit message of my first commit

----------

### Description
Scroll to bottom button style changes to match theme. For light theme it looks as before but for dark it matches the header style.

### CODE REVIEW

**Comment by pprucia** [comment]
Isn't is that the fast scroll works only when you're sending a message? In case user wants to just read messages the button is still useful.

**Comment by moxie0** [comment]
Thanks @pprucia for these kinds of changes can you post before/after images for both light and dark theme? Makes it easier to review, thanks!

**Comment by pprucia** [comment]
screenshot_1487324040
screenshot_1487324076

**Comment by mueller-ma** [comment]
Before:
signal

File: res/drawable/circle_tintable_dark.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:shape="oval">
+    <solid android:color="#bb444444" />
+</shape>
```
Comment (moxie0) [inline res/drawable/circle_tintable_dark.xml]: why do you need a separate shape? if it's white can't you just tint to whatever value you'd want?

**Pull request closed without merging**